### PR TITLE
README: Add a few more words and some installation instructions

### PR DESCRIPTION
**Pull request by issyl0**
- Installation instructions that don't involve `sudo` are always good.
  Now there's a Homebrew formula, so people can install our two tools
  with the same package manager.
- This still needs docs about building for local changes, and releasing,
  but I'm hoping to automate the latter fairly soon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +25,18 @@ sudo gem install govuk-connect
...
+
 ```
+gds govuk connect --help
+```
+
+It can be shortened to `gds govuk c`.
+
+## Developing the gem
+
+TODO: Someone who has made changes to this should write some words here.
```
Comment (benthorner) [inline README.md]: Do we need this? Perhaps we should add it when we've experienced issues with development that merit some docs.

File: README.md [hunk 2]
```diff
@@ -18,6 +25,18 @@ sudo gem install govuk-connect
 
 ## Usage
 
-```bash
-govuk-connect --help
+If you have the gds-cli installed (you [should do](https://docs.publishing.service.gov.uk/manual/access-aws-console.html)!), you can use this tool within it. This means you only have to use one tool to do AWS and SSH.
+
 ```
+gds govuk connect --help
```
Comment (benthorner) [inline README.md]: Since we're offering two installation methods, should we keep both usages?

Comment (issyl0) [reply README.md]: Whichever installation method you use it will still work.

File: README.md [hunk 3]
```diff
@@ -18,6 +25,18 @@ sudo gem install govuk-connect
...
+```
+
+It can be shortened to `gds govuk c`.
+
```
Comment (issyl0) [inline README.md]:
```suggestion
Whichever installation method you choose, you can use the above method or the standalone `govuk-connect` binary.
```

File: README.md (hunk 1 above)
Comment (issyl0) [reply README.md]: Sure.

File: README.md (hunk 2 above)
Comment (benthorner) [reply README.md]: I'm a little confused. What if I do `gds-cli` and I do `gem install govuk-connect`?

File: README.md (hunk 3 above)
Comment (benthorner) [reply README.md]: Ahh, I see 👍

File: README.md (hunk 2 above)
Comment (issyl0) [reply README.md]: `gds` and `gds-cli` are synonymous in my head, maybe that's bad. But they both work. But I expect most people to go for the Homebrew approach as it's easier than getting weird errors about system Ruby versions.

**Comment by issyl0** [comment]
@benthorner I removed the TODOs and added a note about switching between RubyGems and Homebrew. Does that cover it?

File: README.md [hunk 4]
```diff
@@ -8,7 +8,23 @@ govuk-connect ssh -e integration frontend
...
+brew tap alphagov/gds # This will exist if you use the gds-cli
+brew install govuk-connect
+```
+
+> Note: If you previously installed `govuk-connect` via RubyGems, but
+> switched to using Homebrew, you may have to do the following to get
+> it to work:
+> 
+> ```
+> gem uninstall govuk-connect
```
Comment (benthorner) [inline README.md]:
```suggestion
> sudo gem uninstall govuk-connect
```

**Pull request merged**